### ISSUE TITLE
Hand falls down in gazebo 

### ISSUE THREAD

**Issue by ugocupcic** [shadow-robot/sr-ros-interface#87]

**Comment by ugocupcic** [comment]
Would need to attach the hand to a support in default simulation. This is however not trivial: we'd need to create even more urdf files (due to all our different models) + loaders etc...

**Comment by markpitchless** [comment]
Plus exactly what to attach it too is kinda application specific.

**Comment by guihomework** [comment]
To let you know, the rework of the inertia fixes this issue, since the forearm is heavier and with much more inertia than the fingers.
so pull request https://github.com/shadow-robot/sr-ros-interface/pull/138 solves this issue

**Comment by nomadik** [comment]
Hello!

Dear all,

I installed ROS Indigo and Gazebo (that comes by default).
When I installed the hand model from git repasitories, I had problems with dependencies.
When I installed the hand model from debian, the problem with dependencies was solved.

Now I have the problem with the FFJ1 (other **J1) joints, the last joints of the fingers. These joints do not move. I publish data to topic /sh_ffj0_position_controller/command . There are no topics for the ffj1 of ffj2 joints.
Also the movements of the fingers are not smooth. Maybe it is a problem of my installation?
How to fix the poblem with last joints of the fingers?

Bests!
